tensorflow/contrib/lite/kernels/internal/log_quantized_test.cc

```diff
@@ -69,7 +69,7 @@ inline int32 LogPositiveValuesViaFloat(int32 input_val, int input_integer_bits,
 void CheckOutputData(const std::vector<int32>& test_output,
                      const std::vector<int32>& reference_output,
                      const std::vector<int32>& test_input,
-                     const string& check_label, int input_integer_bits,
+                     const std::string& check_label, int input_integer_bits,
```

Comment:
For the string issues, it is probably better to include contrib/lite/string.h

Reply:
OK, will do.